.github/workflows/update_llm_perf_cuda_pytorch.yaml

```diff
@@ -1,9 +1,9 @@
 name: Update LLM Perf Benchmarks - CUDA PyTorch
 
-on:
-  workflow_dispatch:
-  schedule:
-    - cron: "0 0 * * *"
+on: 
+  workflow_dispatch:  # Manual trigger
+  release:            # Trigger on new release
+    types: [published]  
```

Comment:
I don't think this needs commenting

Reply:
Ok, I can remove the comments

Good question. I think it would be more efficient to run the full benchmark with each release of the pip package, rather than on a daily basis. Running it daily seems wasteful, as the hardware remains unchanged, and we’re simply repeating the benchmark for every code change. Since users are likely to benchmark using the PyPI package, it makes more sense to align this workflow with each release. We could also run them manually if we discover any issues with our benchmarks. However, if you prefer running the benchmark daily, I can revert to that schedule. Just let me know your preference

Reply:
i guess there's a misunderstanding. the daily trigger runs different benchmarks (different model+opt+quant) each time because it skips already benchmarked configurations. it is also a way to benchmark all configurations without being limited by the 6 hours time constraint of runners.

Reply:
Thanks for the explanation, it makes much more sense now. I removed the release schedule and left the original one